library/Zend/InputFilter/InputInterface.php

```diff
@@ -14,26 +14,121 @@
 
 interface InputInterface
 {
+    /**
+     * @param bool $allowEmpty
+     *
+     * @return InputInterface
```

Comment:
When the object itself is returned `@return self` is used (same for all other docblock changes in this PR).